syncer/syncer_test.go

```diff
@@ -738,235 +726,6 @@ func (s *testSyncerSuite) TestColumnMapping(c *C) {
 	}
 }
 
-func (s *testSyncerSuite) TestGeneratedColumn(c *C) {
```

Comment:
since `TestGeneratedColumn`  already tested in many it test such as onlineddl sharding and shardddl2,so maybe it's ok to  delete this test directly

Reply:
Maybe we should add UT for `genInsertSQLs/genUpdateSQLs/genDeleteSQLs`. But, it looks too troublesome...

Reply:
SGTM, but i think https://github.com/pingcap/dm/pull/1857  will add some related test